### ISSUE TITLE
Checkpoints are not saved

### ISSUE THREAD

**Issue by cleolotta** [adapter-hub/adapter-transformers#397]
## Environment info
     
- `adapter-transformers` version: 3.0.1
- Platform: 
- Python version: 3.10.4
- PyTorch version (GPU?): 1.12.0 GPU

## Details

Hi, I'm using the run_mlm.py script to train a Bert model with adapter. 
My training takes very long and I would like to resume from a checkpoint as I have limited usage duration on a slurm cluster hat I'm working on. 
For some reason, however, the checkpoints of training are not saved. In an earlier training, I used the same configurations and had extra folders containing the checkpoints from which I could resume training.

Does anyone have an idea what is going wrong?
 
My configurations are: 

--model_name_or_path bert-base-uncased --model_type bert --tokenizer_name bert-base-uncased --evaluation_strategy "steps" --eval_steps 1000 --save_steps 1000 --save_strategy "steps" --cache_dir cache_dir_bert_CDA_gender --train_file ./datasets/augmented-train.txt --validation_file ./datasets/augmented-test.txt --line_by_line True --do_train True --do_eval True --train_adapter True --num_train_epochs 2 --per_device_train_batch_size 16 --per_device_eval_batch_size 16 --fp16 True --adapter_config "pfeiffer+inv" --output_dir ./augmented-model --load_best_model_at_end True --eval_bias True --learning_rate 1e-5 --dataloader_pin_memory True --weight_decay 0.01 --eval_accumulation_steps 10 --gradient_accumulation_steps 8 --overwrite_output_dir False

Thanks very much in advance!

**Comment by adapter-hub-bert** [comment]
This issue has been automatically marked as stale because it has been without activity for 90 days. This issue will be closed in 14 days unless you comment or remove the stale label.

**Comment by adapter-hub-bert** [comment]
This issue was closed because it was stale for 14 days without any activity.
